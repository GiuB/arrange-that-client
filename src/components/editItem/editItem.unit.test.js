import React from 'react';
import Enzyme, {render} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json'

import EditItem from './editItem';

Enzyme.configure({adapter: new Adapter()});

it('EditItem renders', () => {
    const editItem = render(<EditItem
        name='person'
        handleChange={null}
        handleEnter={null}
        handleEsc={null}/>);

    expect(toJson(editItem)).toMatchSnapshot();
});