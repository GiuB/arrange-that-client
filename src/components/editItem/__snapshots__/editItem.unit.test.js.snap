// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditItem renders 1`] = `
<div>
  <div
    class="MuiPaper-root-4 MuiPaper-elevation1-7 MuiPaper-rounded-5 MuiCard-root-3 EditItem-card-1"
  >
    <div
      class="MuiCardHeader-root-31 EditItem-cardHeader-2"
    >
      <div
        class="MuiCardHeader-content-34"
      >
        <h1
          class="MuiTypography-root-37 MuiTypography-headline-42 MuiTypography-alignCenter-51"
        >
          <div
            class="MuiFormControl-root-63"
          >
            <div
              class="MuiInputBase-root-80 MuiInput-root-67 MuiInput-underline-71 MuiInputBase-formControl-81 MuiInput-formControl-68"
            >
              <input
                aria-invalid="false"
                autofocus=""
                class="MuiInputBase-input-90 MuiInput-input-75"
                placeholder="Name"
                type="text"
                value="person"
              />
            </div>
          </div>
        </h1>
      </div>
    </div>
  </div>
</div>
`;
